platforms-serverless-vercel/vercel-cli/run.sh

```diff
@@ -8,18 +8,21 @@ yarn
 export VERCEL_ORG_ID=$VERCEL_ORG_ID
 echo "VERCEL_ORG_ID: $VERCEL_ORG_ID"
 
+# Note: be aware that Vercel truncates logs, so if you add something like `--build-env DEBUG="prisma:*"` plenty of logs will be missing.
+# That will likely influence the "Postinstall hook" check below, which will result in the CI failing with error code `1`.
+
 if [ "$PRISMA_CLIENT_ENGINE_TYPE" == "binary" ]; then
   echo "Binary"
   export VERCEL_PROJECT_ID=$VERCEL_API_BINARY_PROJECT_ID
   echo "VERCEL_PROJECT_ID: $VERCEL_PROJECT_ID"
 
-  yarn -s vercel --token=$VERCEL_TOKEN --build-env DEBUG="prisma:*" --build-env PRISMA_CLIENT_ENGINE_TYPE="binary" --prod --scope=$VERCEL_ORG_ID --confirm --force 1> deployment-url.txt
+  yarn -s vercel --token=$VERCEL_TOKEN --build-env PRISMA_CLIENT_ENGINE_TYPE="binary" --prod --scope=$VERCEL_ORG_ID --confirm --force 1> deployment-url.txt
 else
  echo "Library (Default)"
   export VERCEL_PROJECT_ID=$VERCEL_API_PROJECT_ID
   echo "VERCEL_PROJECT_ID: $VERCEL_PROJECT_ID"
 
-  yarn -s vercel --token=$VERCEL_TOKEN --build-env DEBUG="prisma:*" --build-env PRISMA_CLIENT_ENGINE_TYPE="library" --prod --scope=$VERCEL_ORG_ID --confirm --force 1> deployment-url.txt
+  yarn -s vercel --token=$VERCEL_TOKEN --build-env PRISMA_CLIENT_ENGINE_TYPE="library" --prod --scope=$VERCEL_ORG_ID --confirm --force 1> deployment-url.txt
```

Comment:
Should we try `DEBUG=prisma:get-platform` instead?

Reply:
The `DEBUG` env var was only useful in the discovery phase. No need to add it back unless we have other problems with Vercel at a certain point in the future

Reply:
Maybe add it as a comment? It would not be super obvious to me how to add that when looking just at the file - a comment would help.

Reply:
There's already a comment at the top of the file, lines 11-12:

```bash
# Note: be aware that Vercel truncates logs, so if you add something like `--build-env DEBUG="prisma:*"` plenty of logs will be missing.
# That will likely influence the "Postinstall hook" check below, which will result in the CI failing with error code `1`.
```

Am I missing something else? @janpio